src/main/webapp/WEB-INF/jsp/application/vacation_overview.jsp

```diff
@@ -44,122 +47,233 @@
 
 <uv:menu/>
 
-<div class="content print--only-portrait">
+<div class="content">
     <div class="container">
 
         <c:if test="${IS_ALLOWED}">
 
-            <div class="row">
+            <div class="row print:mb-4">
                 <div class="col-xs-12">
                     <legend id="vacation">
                         <spring:message code="overview.vacationOverview.title"/>
```

Comment:
No print button displayed here, maybe this could be useful?
```suggestion
                        <spring:message code="overview.vacationOverview.title"/>
                        <uv:print/>
```